plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -154,6 +211,7 @@ def test_jobs_configured_via_cmd_and_config(hydra_sweep_runner: TSweepRunner) ->
         assert len(returns) == 2
         best_parameters = returns.ax
         assert math.isclose(best_parameters["quadratic.x"], -2.0, abs_tol=1e-4)
+        # TODO: y should actually be -1
```

Comment:
Yes, if you look at the optimization problem, the solution is y=-1 in this case. Since the sweeper only does a few tries, it happens to output +1. I don't think this is the right way to test.

Reply:
The tests have been stable  and it would be hard to predict the sweeper's behavior beyond a couple of steps so the tests make the best efforts here.

Please do feel free to suggest alternatives via a issue or a PR. Adding a TODO here makes it harder to track.  

Thanks!